README.md

```diff
@@ -14,10 +14,55 @@ Welcome to NI's internal and external C# conventions and linter rules/plugins/to
 
 ## Quickstart
 
-Add a package reference to the package TODO_TBD.
+### Package Source
+Under the `packageSources` node, add a reference to the source of the package in your
+[`NuGet.Config`](https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior) file.
+
+```xml
+<packageSources>
+  <add key="<todo-tbd>" value="<todo-tbd>" />
+</packageSources>
+```
+
+### Package Reference
+Add a package reference to the package `NI.CSharp.Analyzers`.
 
 ```msbuild
-<PackageReference Include="" Version="" />
+<PackageReference Include="NI.CSharp.Analyzers" Version="0.1.0" />
+```
+
+**Note**: Currently, this package only works for projects that reference it using
+[`PackageReference`](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files).
```

Comment:
You should mention that packages.config doesn't work explicitly

Reply:
Done, fixed in the latest update.